src/api/vif.js

```
@@ -85,16 +84,16 @@ export async function set ({
     attached == null && (attached = vif.attached)
 
     await this.allocIpAddresses(vif.id, null, oldIpAddresses)
+    await xapi.deleteVif(vif._xapiId)::pCatch(noop)
```

Do not ignore errors.